app/CacheProvider.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\SearchService;
+
+use Predis\Client;
+use Symfony\Component\Cache\Adapter\RedisAdapter;
+
+final class CacheProvider extends BaseServiceProvider
+{
+    protected $provides = [
+        RedisAdapter::class,
+    ];
+
+    public function register(): void
+    {
+        $this->add(
+            RedisAdapter::class,
+            fn (): RedisAdapter =>
+            new RedisAdapter(
+                new Client(
+                    $this->parameter('cache.redis')
+                ),
+                'permission' . '_',
+                86400,
+            ),
+        );
```

Comment:
```suggestion
    protected $provides = [
        CacheInterface::class,
    ];

    public function register(): void
    {
        $this->add(
            CacheInterface::class,
            fn (): RedisAdapter =>
            new RedisAdapter(
                new Client(
                    $this->parameter('cache.redis')
                ),
                'permission' . '_',
                86400,
            ),
        );
```

it could make sense to depend on the interface that the adapters adhere to. So that you could swap out the implementation.

Reply:
This was fixed, can you approve @simon-debruijn